pyproject.toml

```
@@ -59,7 +59,10 @@ test = [
 	# for tools/finalize.py
 	'jaraco.develop >= 7.21; python_version >= "3.9" and sys_platform != "cygwin"',
 	"pytest-home >= 0.5",
-	"mypy==1.10.0", # pin mypy version so a new version doesn't suddenly cause the CI to fail
+	# pin mypy version so a new version doesn't suddenly cause the CI to fail,
+	# until types-setuptools is removed from typeshed.
+	# For help with static-typing issues, or mypy update, ping @Avasam 
+	"mypy==1.11",
```

This pin is the same as ==1.11.0. Is that what you intended, or did you want something like `~=1.11` or `==1.11.*`?

---

Yes. Seems I mixed up with npm
```suggestion
	"mypy==1.11.*",
```